app/lib/common/themes/app_theme.dart

```diff
@@ -39,257 +40,32 @@ class EmojiConfig {
   static final emojiSizeMax = 32 * ((!kIsWeb && Platform.isIOS) ? 1.30 : 1.0);
 }
 
-AppTheme currentTheme = AppTheme();
-
 extension CustomColorScheme on ColorScheme {
-  Color get success => const Color(0xFF67A24A);
-  Color get tertiary2 => const Color(0xFFFFC333);
+  Color get primary => const Color(0xFF9CCAFF);
+  Color get tertiary => const Color(0xFFFFC333);
   Color get tertiary3 => const Color(0xFF3AE3E0);
   Color get neutral => const Color(0xFF121212);
   Color get neutral2 => const Color(0xFF2F2F2F);
   Color get neutral3 => const Color(0xFF5D5D5D);
   Color get neutral4 => const Color(0xFF898989);
   Color get neutral5 => const Color(0xFFB7B7B7);
   Color get neutral6 => const Color(0xFFE5E5E5);
-  Color get m3Primary => const Color(0xFFD0BCFF);
-  Color get navBarBackground => const Color(0xFF122D46);
 
-  Color get error => const Color(0xFFFF0000);
+  // brand
+  Color get textHighlight => secondary;
+  Color get textButtonColor => yellowColor;
+
+  // states
+  Color get success => secondary;
 
-  Color get badgeUnread => const Color(0xFF67A24A);
-  Color get badgeImportant => const Color(0xFFFFC333);
+  // specific widgets
+  Color get badgeUnread => secondary;
```

Comment:
Where is `secondary` color declaration? Are we considering default `ThemeData` secondary color?

Reply:
yes, this directly comes from the regular ThemeData. That's effectivly what the get does, just creates an alias for the more semantic `badgeUnread` color...